src/actions/project-text-search.js

```diff
@@ -35,6 +36,13 @@ export function clearSearchResults() {
   };
 }
 
+export function clearSearch() {
```

Comment:
Rather than dispatching two actions couldn't we have one action that is listened to in the reducer and does the work?

Reply:
sure.. I'll hook in a proper action in place.